src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -268,6 +270,13 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
         datasetId = DatasetId.of(project, normalizedDatabaseName);
         primaryKeyStore.clear();
         if (bigQuery.getDataset(datasetId) != null) {
+          if (requireManualDrops) {
+            throw new IllegalArgumentException(
```

Comment:
Nit, should we throw general RuntimeException rather than IllegalArgumentException, seems like the case does not like an illegal argument case.